mobile/apps/auth/lib/ui/settings/data/local_backup_settings_page.dart

```diff
@@ -1,452 +1,259 @@
-
-                    Expanded(
-                      child: ButtonWidget(
-                        buttonType: ButtonType.primary,
-                        labelText: l10n.saveAction,
-                        isDisabled: textController.text.isEmpty,
-                        onTap: () async => Navigator.of(context).pop(textController.text),
-                      ),
-                    ),
-                  ],
+                Padding(
+                  padding: const EdgeInsets.only(top: 12, left: 12, right: 12),
+                  child: Text(
+                    l10n.localBackupDailyManualCopy,
+                    style: textTheme.miniFaint,
+                  ),
+                ),
+                const SizedBox(height: 28),
+                AnimatedSwitcher(
+                  duration: const Duration(milliseconds: 250),
+                  switchInCurve: Curves.easeOut,
+                  switchOutCurve: Curves.easeIn,
+                  child: controller.isBackupEnabled
+                      ? Column(
+                          crossAxisAlignment: CrossAxisAlignment.start,
+                          children: [
+                            FutureBuilder<bool>(
+                              future: controller.hasPasswordConfigured(),
+                              builder: (context, snapshot) {
+                                final hasPassword = snapshot.data ?? false;
+                                final tileTitle = hasPassword
+                                    ? 'Update backup password'
+                                    : 'Set backup password';
+                                return MenuItemWidget(
+                                  captionedTextWidget: CaptionedTextWidget(
+                                    title: tileTitle,
+                                  ),
+                                  singleBorderRadius: 12,
+                                  alignCaptionedTextToLeft: true,
+                                  menuItemColor: colorScheme.fillFaint,
+                                  trailingIcon: Icons.chevron_right_outlined,
+                                  trailingIconIsMuted: true,
+                                  onTap: () async {
+                                    await controller.updatePassword(context);
+                                  },
+                                );
+                              },
+                            ),
+                            const SizedBox(height: 20),
+                            _BackupLocationCard(
+                              controller: controller,
+                              colorScheme: colorScheme,
+                              textTheme: textTheme,
+                            ),
+                            const SizedBox(height: 12),
+                            _BackupButton(
+                              onPressed: controller.runManualBackup,
+                              label: 'Backup now (manual)',
```

Comment:
Same. can check it for other places also (if any)

Reply:
Fixed!